docs/prow_plugins_additional_tooling.md

```diff
@@ -0,0 +1,234 @@
+## Overview
+
+Investigation results for the plugins and additional tooling used by Prow. This investigation was done on 12 September 2018.
+
+### Prow Plugins[1]
+
+**approve**
```

Comment:
If this is just a copy/paste from https://prow.k8s.io/plugins IMO it does not make sense to keep it here, link will be enough.